cmd/utils/flags.go

```diff
@@ -591,6 +592,57 @@ var (
 		Name:  "anchoring",
 		Usage: "Enable anchoring for service chain",
 	}
+	// ChainDataFetcher
+	EnableChainDataFetcherFlag = cli.BoolFlag{
+		Name:  "chaindatafetcher",
+		Usage: "Enable the ChainDataFetcher Service",
+	}
+	ChainDataFetcherNoDefault = cli.BoolFlag{
+		Name:  "chaindatafetcher.no.default",
+		Usage: "Turn off the starting of the chaindatafetcher",
+	}
+	ChainDataFetcherNumHandlers = cli.IntFlag{
+		Name:  "chaindatafetcher.num.handlers",
+		Usage: "Number of chaindata handlers",
+		Value: 10,
+	}
+	ChainDataFetcherReqChSize = cli.IntFlag{
+		Name:  "chaindatafetcher.request.buffer",
+		Usage: "Buffer size of request channel",
+		Value: 50,
+	}
+	ChainDataFetcherResChSize = cli.IntFlag{
+		Name:  "chaindatafetcher.response.buffer",
+		Usage: "Buffer size of response channel",
+		Value: 50,
+	}
```

Comment:
How about using one variable for channels?

Reply:
I merged two flags into one.